### DISCUSSION TITLE
Hiding certain endpoints - Railway hosted app?

### DISCUSSION THREAD

**Discussion by Unlok-Simon** [danny-avila/LibreChat#3256] [Q&A] [answered]
Hi

I have deployed a hosted app to Railway which is working great. However, I want to not display most of the endpoints in the selection dropdown as they won't be needed. Reading through the documentation I thought this would happen if I changed the API key envrionment variable to empty (from user chosen). I tried this for Fireworks but the endpoint is still visible.

I also checked the Endpoints variable but it only contains a small list compared to everything available for selection.

What am I missing?

Thanks

**Comment by fuegovic** [comment]
Removing the value of `CONFIG_PATH` will disable all "custom" endpoints

**Reply by berry-13** [reply]
yea, you could simply remove the value from the 'CONFIG_PATH', but keep in mind that this might not be the best solution as future features will rely on it

**Comment by berry-13** [comment, marked as answer]
If you want to use a specific endpoint, you'll need to create your own `librechat.yaml` file. You can host it on GitHub and add the link to your "CONFIG_PATH" (avoid hardcoding your API keys). If you're unsure, you can look at an example here: https://raw.githubusercontent.com/fuegovic/lc-config-yaml/main/librechat-rw.yaml. Have a look at the documentation to learn how to customize librechat

![image](https://github.com/danny-avila/LibreChat/assets/81851188/1810035b-cd7f-4cce-b0f6-891cdda456be)

**Reply by Unlok-Simon** [reply]
Looks like the custom CONFIG_PATH is the way to go. Thanks

It would be nice that if an endpoint is in the envrionment variables and you remove its API value it simply doesn't display it in the UI.

**Reply by berry-13** [reply]
This applies to all endpoints except the custom ones, as you would have specified those in your file
